### DISCUSSION TITLE
Button shape standard?

### DISCUSSION THREAD

**Discussion by chris480** [Myndex/SAPC-APCA#93] [Q&A] [answered]
Reading through [Discussion #39](https://github.com/Myndex/SAPC-APCA/discussions/39) I'm not sure what button shape refers? Is there a defined standard button shape?

Referring to this graphic 
![image](https://user-images.githubusercontent.com/42009457/136927662-5a40919d-5a42-4cbb-a965-445489b37950.png)

**Comment by Myndex** [comment, marked as answer]
Hi Chris @chris480 

> _I'm not sure what button shape refers? Is there a defined standard button shape?_

No defined shape, I think what you are referring to is the spatial frequency of the button. Generally, it is recommended to have a minimum of 0.5em or 1ex of padding around the text and the button edge. 1em or 2ex is preferred padding.
